### PR TITLE
Release/snowplow web/1.0.0

### PR DESCRIPTION
**Pull request by rlh1994**
## Description & motivation
This marks the 1.0.0 release of web. The changes are mostly cosmetic or a few small fixes, hence the single PR. 

## Checklist
- [x] I have verified that these changes work locally
- [x] I have updated the README.md (if applicable)
- [x] I have added tests & descriptions to my models (and macros if applicable)
- [x] I have raised a [documentation](https://github.com/snowplow/documentation) PR if applicable (Link [here](https://github.com/snowplow/documentation/pull/692) if required)
- [x] Is your change a breaking change?

## Release Only Checklist
- [x] I have updated the version number in all relevant places
- [x] I have updated the CHANGELOG.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +74,7 @@ This distribution is all licensed under the [Snowplow Community License, Version
...
+[maintained]: https://img.shields.io/static/v1?style=flat&label=Snowplow&message=Maintained&color=a069d7&labelColor=9ba0aa&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAeFBMVEVMaXGXANeYANeXANZbAJmXANeUANSQAM+XANeMAMpaAJhZAJeZANiXANaXANaOAM2WANVnAKWXANZ9ALtmAKVaAJmXANZaAJlXAJZdAJxaAJlZAJdbAJlbAJmQAM+UANKZANhhAJ+EAL+BAL9oAKZnAKVjAKF1ALNBd8J1AAAAKHRSTlMAa1hWXyteBTQJIEwRgUh2JjJon21wcBgNfmc+JlOBQjwezWF2l5dXzkW3/wAAAHpJREFUeNokhQOCA1EAxTL85hi7dXv/E5YPCYBq5DeN4pcqV1XbtW/xTVMIMAZE0cBHEaZhBmIQwCFofeprPUHqjmD/+7peztd62dWQRkvrQayXkn01f/gWp2CrxfjY7rcZ5V7DEMDQgmEozFpZqLUYDsNwOqbnMLwPAJEwCopZxKttAAAAAElFTkSuQmCC
 [tracker-docs]: https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/
 [webpage-context]: https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/tracker-setup/initialization-options/#adding-predefined-contexts
 [dbt-package-docs]: https://docs.getdbt.com/docs/building-a-dbt-project/package-management
```
Comment (emielver) [inline README.md:80]: The discourse shield is saying weird things but not sure we can do anything about it

**Review by emielver — APPROVED** [review]
Looks good!

File: README.md (hunk 1 above)
Comment (rlh1994) [reply README.md:80]: I think it's broken everywhere, I tried their docs with the dbt discourse and seems to also fail for that url so....

**Review by georgewoodhead — APPROVED** [review]
LGTM!!

**Pull request merged**